mne/annotations.py

```diff
@@ -295,6 +316,18 @@ def save(self, fname):
             with start_file(fname) as fid:
                 _write_annotations(fid, self)
 
+    def _sort(self):
+        """Sort in place."""
+        # instead of argsort here we use sorted so that it gives us
+        # the onset-then-duration hierarchy
+        sorter = [(o, d, i) for o, d, i in zip(self.onset,
+                                               self.duration,
+                                               range(len(self)))]
```

Comment:
why do we need to state the tuple?

```py
In [12]: [_ for _ in zip(('a1', 'a2', 'a3'),('b1', 'b2', 'b3'),range(3))]                               
Out[12]: [('a1', 'b1', 0), ('a2', 'b2', 1), ('a3', 'b3', 2)]
```

I guess that `sorter = [ _ for _ in zip(self.onset, self.duration, range(len(self)))]` should be enough.

Reply:
```py
In [26]: sorted(zip(('a3', 'a2', 'a1'),('b3', 'b2', 'b1'),range(3)))                                                                        
Out[26]: [('a1', 'b1', 2), ('a2', 'b2', 1), ('a3', 'b3', 0)]
```

Therefore
```py
sorter = sorted(zip(self.onset, self.duration, range(len(self))))
```